app/services/streamlabels/definitions.ts

```diff
@@ -555,26 +548,109 @@ const youtubeDefinitions: IStreamlabelSet = {
 const twitchDefinitions: IStreamlabelSet = {
   // Trains are currently disabled
```

Comment:
Looks like this comment is not actual now

Reply:
Oops, will remove